pulp_2_tests/tests/rpm/cli/test_sync.py

```diff
@@ -202,6 +207,44 @@ def _list_units(cfg, unit_type):
         )).stdout.splitlines()
 
 
+class ForceSyncDuplicateSRPMFailure(_BaseTestCase):
```

Comment:
If it is not really necessary, we should avoid call clean orphans. It is an expensive call.
```suggestion
class ForceSyncDuplicateSRPMFailure:
```